services/site/src/pages/index.tsx

```diff
@@ -80,67 +68,78 @@ const Home: React.VoidFunctionComponent<HomeProps> = ({ fmType }) => {
         <meta name="theme-color" content="#FFFFFF" media="(prefers-color-scheme: light)" />
       </Head>
-      <Navigation displayBreadcrumbs />
-      <main>
-        <h1 className={clsx("sr-only")} aria-label="Allyphant">
-          a11yphant
-        </h1>
-        <div className={clsx("w-full h-full")}>
-          {!currentUser?.isRegistered && <Hero />}
-          <section
-            id="challenges"
-            className={clsx("max-w-screen-3xl pt-10 mx-8 mt-22 mb-4", "sm:mx-12 sm:mt-18 sm:mb-12", "md:mx-24 md:mb-24", "2xl:mx-auto")}
-          >
-            <ChallengeHeader className={clsx("2xl:mx-24")} userLoggedIn={currentUser?.isRegistered} />
-
-            <Legend className={clsx("2xl:mx-24")} />
-            <p className={clsx("mb-12 text-grey-middle", "2xl:mx-24")}>
-              <span className="sr-only">Disclaimer:</span>
-              <span aria-hidden="true" className="text-xl text-grey-middle">
-                *
-              </span>{" "}
-              Currently, we only offer easy challenges. However, our small team is dedicated to producing more content for you. We will announce new
-              challenges on our{" "}
-              <InTextLink
-                href="https://twitter.com/a11yphant"
-                opensInNewTab
-                overrideClassName
-                className="text-grey-middle font-sans font-normal border-grey-middle
-        transition-colors duration-300
-        hover:text-primary-light hover:border-transparent
-        focus-rounded-instead-of-underline"
-              >
-                Twitter account
-              </InTextLink>
-              . <br />
-            </p>
-
-            {dataChallengesInProgress?.challenges.length > 0 && (
-              <ChallengeList
-                className={clsx("2xl:mx-24")}
-                heading={"Continue where you left"}
-                challenges={dataChallengesInProgress.challenges}
-                displayCompleted={false}
-              />
-            )}
-
-            {easyChallenges.length > 0 && (
-              <ChallengeList
-                className={clsx("2xl:mx-24")}
-                heading={dataChallengesInProgress?.challenges.length === 0 ? "All challenges" : "Other challenges"}
-                challenges={easyChallenges}
-              />
+      <Navigation />
+      <main className="bg-texture bg-repeat-y bg-contain bg-origin-border bg-top">
+        <div className={clsx("h-full max-w-screen-3xl px-8", "sm:px-12", "md:px-24", "xl:px-24", "2xl:mx-auto")}>
+          <HeroSection />
+          <MediaSection />
+          <TopChallengeSection>
+            {topChallenges.data && (
+              <>
+                <TopChallenge challenge={topChallenges.data.challenge1} timesCompleted="3400" isUserFavorite />
+                <TopChallenge challenge={topChallenges.data.challenge2} timesCompleted="1700" />
+                <TopChallenge challenge={topChallenges.data.challenge3} timesCompleted="60" isNew />
```

Comment:
```suggestion
                <TopChallenge challenge={topChallenges.data.challenge1} timesCompleted="1350" isUserFavorite />
                <TopChallenge challenge={topChallenges.data.challenge2} timesCompleted="650" />
                <TopChallenge challenge={topChallenges.data.challenge3} timesCompleted="25" isNew />
```

The numbers aren't correct.

Also, 25 isn't very impressive. Maybe replace the number with "Recently added" for this challenge?

Reply:
@thomasdax98 Where did you get the data from? I used the most clicked level within a challenge.

Reply:
You used the number of tries for the first level. There are two issues with that:

### 1) Tries vs Success

Try = Submission with unknown outcome (could be success, failed or error)

The number of tries is the number of times the level was submitted for evaluation regardless of the outcome (could be success, failed or error). One user can need multiple tries (e.g. if they fail first, then submit a second time which is successful) before they complete the level successfully. 

Success = Successful submission

The number of successes is the number of times the level was completed successfully.

### 2) Completing one level vs completing the challenge

The card states "3400+ TIMES COMPLETED". But you are using the number of users who started the challenge, not who completed the challenge.

In order to complete a challenge, a user needs to complete all levels of the challenge. So the number of users who completed a challenge can be approximated using the number of successful submissions for the level with the least successful submissions.

In the case of the "A valid HTML document challenge", level 1 has the most successes (2103) and level 7 has the least successes (1274). 
The difference (829) is the amount of users we lost along the way.

(I actually just realised that the numbers I used above are also slightly off. I just corrected them)